src/web/components/MostViewed/MostViewedRight/MostViewedRight.tsx

```diff
@@ -31,17 +31,19 @@ export const MostViewedRight = ({ pillar, limitItems = 5 }: Props) => {
     }
 
     if (data) {
+        // Look I don't know why data-component is geo-most-popular either, but it is, ok? Ok.
         return (
-            <div className={wrapperStyles}>
+            <div className={wrapperStyles} data-component="geo-most-popular">
                 <GuardianLines pillar={pillar} count={4} />
                 <h3 className={headingStyles}>most viewed</h3>
-                <ul>
+                <ul data-link-name="Right hand most popular geo GB">
                     {(data.trails || [])
                         .slice(0, limitItems)
                         .map((trail: TrailType, ii: number) => (
                             <MostViewedRightItem
                                 key={trail.url}
                                 trail={trail}
+                                ii={ii}
```

Comment:
What does `ii` do? I presume it's the index but I can't tell from this name why it's being passed down.

Reply:
Fixed 👍